package.json

```diff
@@ -10,8 +10,8 @@
   "scripts": {
     "dev": "webpack server",
     "build": "webpack",
-    "test": "npx jest",
-    "coverage": "jest --coverage --silent",
+    "test": "npx jest -u",
+    "coverage": "jest --coverage --silent -u",
```

Comment:
### Critical 

Same here, restore the previous command. You have to be sure to update snapshot before you push the code not when running the github actions